formats/json/jsMain/src/kotlinx/serialization/json/internal/FormatLanguage.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.json.internal;
+
+import kotlinx.serialization.InternalSerializationApi
+
+@InternalSerializationApi
+public actual typealias FormatLanguage = org.intellij.lang.annotations.Language
```

Comment:
Class `org.intellij.lang.annotations.Language` is declared in a neighboring file, so that's enough.

However, as I understand it, I will have to make a separate compile-only module because of JPMS.

Reply:
Sorry, I've commented wrong file. I can see that `public actual typealias FormatLanguage = org.intellij.lang.annotations.Language` is in the jvmMain, but I do not see `org.intellij.lang.annotations.Language` in `jvmMain` anywhere.

Reply:
Because `org.jetbrains:annotations` is a transitive dependency from JVM stdlib.